bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,50 @@
+from discord.ext import commands
+from pathlib import Path
+from discord import Embed
+import random
+import json
+
+embeds = []
+
+# cog
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    def get_response(self) -> Embed:
+        return random.choice(embeds)
+
+    @commands.command(aliases=('save_the_earth',))
```

Comment:
I'm not quite enjoying the command name. Can we make it either one word (.savetheplanet) which can be shortened for aliases (.saveplanet / .saveearth), or a group with subcommands (.save planet / .save earth)?

Reply:
OK, I'll remove the underscores.